packages/appstore/src/assets/svgs/wallets/wallets-image.tsx

```diff
@@ -31,7 +34,10 @@ export const WalletsImagesList = {
     upgrade_mobile: UpgradeMobileImage,
     upgrading_mobile: ReadyMobileImage,
     upgrading_mobile_eu: ReadyMobileEuImage,
-    ready_to_upgrade_wallets_image: ReadyToUpgradeWalletsImage,
+    ready_to_enable_wallets_desktop: ReadyToEnableWalletsDesktop,
+    ready_to_enable_wallets_mobile: ReadyToEnableWalletsMobile,
+    why_wallets_desktop: WhyWalletsDesktopImage,
+    why_wallets_mobile: WhyWalletsMobileImage,
```

Comment:
Small suggestion: considering that the components were renamed to wallets-upgrade-step-one and wallets-upgrade-step-two wouldn't it make the flow and usage more clear if we were to match the assets and svg files to that naming convention (i.e WalletsUpgradeStepOneDesktop or UpgradeStepOneDesktopImage)

Reply:
Done